core/src/main/scala/water/support/DeepLearningSupport.scala

```diff
@@ -42,6 +44,7 @@ trait DeepLearningSupport {
     * @tparam T H2O Frame Type
     * @return Deep Learning Model
     */
+  @DeprecatedMethod("Sparkling Water Algorithm API to train and score H2O models", "3.32")
```

Comment:
Maybe, I would add reference to the `H2ODeepLearning` class.

Reply:
👍